content/200-orm/200-prisma-client/300-client-extensions/500-middleware/index.mdx

```diff
@@ -8,7 +8,7 @@ metaDescription: 'Prisma Client middleware allows you to perform actions before
 
 <Admonition type="warning">
 
-**Deprecated**: Middleware is deprecated in version 4.16.0.
+Middleware has been removed in [v6.14.0](https://pris.ly/release/6.14.0) and had been deprecated since [v4.16.0](https://github.com/prisma/prisma/releases/tag/4.16.0).
```

Comment:
```suggestion
**Middleware has been removed in [v6.14.0](https://pris.ly/release/6.14.0)** and had been deprecated since [v4.16.0](https://github.com/prisma/prisma/releases/tag/4.16.0).
```